cpp/tests/CMakeLists.txt

```diff
@@ -617,6 +617,11 @@ ConfigureTest(
 # * bin tests ----------------------------------------------------------------------------------
 ConfigureTest(LABEL_BINS_TEST labeling/label_bins_tests.cpp)
 
+# ##################################################################################################
+# * jit tests ----------------------------------------------------------------------------------
+ConfigureTest(JIT_PARSER_TEST jit/parse_ptx_function.cpp)
+target_include_directories(JIT_PARSER_TEST PRIVATE "$<BUILD_INTERFACE:${CUDF_SOURCE_DIR}/src>")
```

Comment:
What do we need this include for? What header is getting pulled?

Reply:
The `<jit/parser.hpp>` file includes other headers that are private to cudf. So I needed this include to test the JIT parser in isolation.